universal-application-tool-0.0.1/app/assets/stylesheets/styles.css

```diff
@@ -20,4 +20,7 @@
   button:hover {
     @apply bg-blue-700;
   }
+  main {
+    @apply max-w-screen-xl mx-auto;
+  }
```

Comment:
Nit. I don't love the lack of blank lines between blocks, but if that's the standard...

Reply:
Done